hrui/tests/phpunit/CRM/HRUI/HelperTest.php

```diff
@@ -12,36 +14,57 @@ class CRM_HRUI_HelperTest extends \PHPUnit_Framework_TestCase implements Headles
 
   use HRUITrait;
 
+  protected $requiredExtensions = [
+    'uk.co.compucorp.civicrm.hrcore', // required for fabricator
+    'org.civicrm.hrident', // creates the Identity custom group required by hrcase
+    'uk.co.compucorp.civicrm.tasksassignments', // if not enabled will try to redirect
+    'org.civicrm.hrcase' // creates the line manager relationship type
+  ];
+
+  /**
+   * @return CiviEnvBuilder
+   */
   public function setUpHeadless() {
-    // hrcase create ( Line Manager is ) relationship type which is need for the tests
     return \Civi\Test::headless()
+      ->install($this->requiredExtensions)
       ->installMe(__DIR__)
-      ->install('org.civicrm.hrcase')
       ->apply();
   }
 
+  /**
+   * Checks that the CRM_HRUI_Helper returns all line managers
+   */
   public function testGetLineManagersList() {
-    $contactParamsA = array("first_name" => "chrollo", "last_name" => "lucilfer");
-    $contactParamsB = array("first_name" => "hisoka", "last_name" => "morou");
-    $contactA = $this->createContact($contactParamsA);
-    $contactB = $this->createContact($contactParamsB)
-    ;
-    $this->createRelationship($contactA, $contactB, 'Line Manager is');
+    $relationshipType = 'Line Manager is';
+
+    $contactA = $this->createContact('chrollo', 'lucilfer');
```

Comment:
It's purely to keep things under 80 characters, the alternatives are removing the `createContact` method and using

```
   $contactA = ContactMaker::fabricate(['first_name' => 'chrollo', 'last_name' => 'lucilfer'])['id'];
```

Which is 103 chars with whitespace. Or:

```
    $params = ['first_name' => 'chrollo', 'last_name' => 'lucilfer'];
    $contactA = ContactMaker::fabricate($params)['id'];
```

Which is short, but takes two lines each time and the array structure is duplicated.

I agree with your point that it's less future proof but think that the method makes things more readable for now and it should be easy to change if we ever need to.